src/common/common.gyp

```diff
@@ -0,0 +1,26 @@
+{
+  'includes':[
+    'common.gypi',
+  ],
+  'targets': [{
+    'target_name': 'tizen_common',
+    'type': 'static_library',
+    'sources': [
+      'extension.cc',
+      'extension.h',
+      'logger.cc',
+      'logger.h',
+      'picojson.h',
+      'scope_exit.h',
+      'utils.h',
+      'XW_Extension.h',
+      'XW_Extension_EntryPoints.h',
+      'XW_Extension_Permissions.h',
+      'XW_Extension_Runtime.h',
+      'XW_Extension_SyncMessage.h',
+    ],
```

Comment:
Thanks, I saw it [here](https://github.com/crosswalk-project/tizen-extensions-crosswalk/blob/master/src/media_renderer/media_renderer.gyp#L176). And this, is in fact, main issue here. This flag is removed on bottom of GYP inclusion tree so we do not have place to turn it back on for included "src/common" files. 

This is why I decided to move common sources to different target and include only where it's needed.

Other solution is to compile those additional targets in media_renderer, mediaserver and speech with C++11 (I tried and it's compiling without errors). But I assume we should not touch this '-std=c++0x' flag removal because it's dbus_proxy related stuff. I'm guessing that it will brake something in runtime.